src/frontend/src/pages/part/PartPricingPanel.tsx

```diff
@@ -0,0 +1,115 @@
+import { t } from '@lingui/macro';
+import { Accordion, Alert, LoadingOverlay, Stack, Text } from '@mantine/core';
+import { ReactNode, useMemo } from 'react';
+
+import { ApiEndpoints } from '../../enums/ApiEndpoints';
+import { UserRoles } from '../../enums/Roles';
+import { useInstance } from '../../hooks/UseInstance';
+import { useUserState } from '../../states/UserState';
+import BomPricingPanel from './pricing/BomPricingPanel';
+import PriceBreakPanel from './pricing/PriceBreakPanel';
+import PricingOverviewPanel from './pricing/PricingOverviewPanel';
+import PricingPanel from './pricing/PricingPanel';
+import PurchaseHistoryPanel from './pricing/PurchaseHistoryPanel';
+import SaleHistoryPanel from './pricing/SaleHistoryPanel';
+import SupplierPricingPanel from './pricing/SupplierPricingPanel';
+import VariantPricingPanel from './pricing/VariantPricingPanel';
+
+export default function PartPricingPanel({ part }: { part: any }) {
+  const user = useUserState();
+
+  const {
+    instance: pricing,
+    refreshInstance,
+    instanceQuery
+  } = useInstance({
+    pk: part?.pk,
+    hasPrimaryKey: true,
+    endpoint: ApiEndpoints.part_pricing_get,
+    defaultValue: {}
+  });
+
+  // TODO: Do we display internal price? This is a global setting
+  const internalPricing = true;
```

Comment:
Comment: Settings can be accessed via useGlobalSettingsState zustand

Reply:
Done now, that was a placeholder for myself :)